lib/much-rails/date.rb

```diff
@@ -0,0 +1,48 @@
+module MuchRails
+  # @example
+  #   MuchRails::Date(nil) # => nil
+  #   MuchRails::Date(" ") # => nil
+  #   MuchRails::Date(Time.zone.today) # => Date
+  #   MuchRails::Date(Time.current) # => Date
+  #   MuchRails::Date(DateTime.current) # => Date
+  #   MuchRails::Date("07/04/2020") # => Date
+  #   MuchRails::Date("2020.07.04") # => Date
+  #   MuchRails::Date("2020-07-04T08:15:00Z") # => Date
+  def self.Date(value)
```

Comment:
can we switch to this `Romo::Date.for(value)` instead? Sorry forgot to mention this earlier.